app/src/main/java/org/imozerov/streetartview/storage/model/Extensions.kt

```diff
@@ -17,7 +17,6 @@ fun RealmArtObject.copyDataFromJson(artwork: Artwork) {
             val realmAuthor = RealmAuthor()
             realmAuthor.id = it.id
             realmAuthor.name = it.name
-            realmAuthor.photo = it.photo
```

Comment:
Do we no longer need author's photo? If so it gonna be replaced from server too.